client/src/lib.rs

```diff
@@ -1441,4 +1471,27 @@ mod tests {
         let next_req = iceptor.call(req).unwrap();
         assert_eq!(next_req.metadata().get("enchi").unwrap(), "cat");
     }
+
+    #[test]
+    fn keep_alive_defaults() {
```

Comment:
I dunno how useful this is really, but, since you wrote it I'm fine keeping it too.

Reply:
I was scared of my Rust string in the default builder and I wanted to confirm it worked just for me, so I figured why not a test, but I agree in general this is a silly test